source/crud/tests/v2/default-write-concern.yml

```diff
@@ -0,0 +1,413 @@
+data:
+  - _id: 1
+    x: 11
+  - _id: 2
+    x: 22
+collection_name: default_write_concern_coll
+database_name: default_write_concern_db
+tests:
+  - description: Aggregate with $merge
```

Comment:
I think this will need to be moved to a separate test file, since [`$merge`](https://docs.mongodb.com/manual/reference/operator/aggregation/merge/) was introduced in 4.2. For simplicity, you may want to have a common `default-write-concern.yml` test file and then split certain tests with `minServerVersion` requirements into separate files (e.g. `default-write-concern-4.2.yml`).

The basic insert, update, and delete commands wouldn't need a `minServerVersion`, but you'll definitely want to split files findAndModify (3.2+) and other write commands (3.4+), per https://github.com/mongodb/specifications/blob/master/source/wireversion-featurelist.rst.

Reply:
Done.